packages/peid.vm/peid.vm.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>peid.vm</id>
-    <version>0.95</version>
+    <version>0.95.20081103</version>
```

Comment:
I have used the tool official date. I could also use today's date, but I though that looks nicer and as the package hasn't been pushed yet, it is not really important.

Reply:
@Ana06 Make sure to only put dates in the fourth segment 😄